projects/canopy/src/lib/card/card-hero-img/card-hero-img.component.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+  ViewEncapsulation,
+} from '@angular/core';
+
+@Component({
+  selector: 'lg-card-hero-img',
+  templateUrl: './card-hero-img.component.html',
+  styleUrls: [ './card-hero-img.component.scss' ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
+})
+export class LgCardHeroImageComponent {
+  @Input() coverImageUrl: string;
+  @Input() imageSrc: string;
+  @Input() imageAlt = 'card-hero-img';
```

Comment:
The `alt` should be an empty `string` by default. The screen reader wouldn't read the `card-hero-img` nicely and also it wouldn't tell anything interesting/useful to the user 😅

Reply:
Done. :)